javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var theForm = document.querySelector('#theForm');
```

Comment:
Make sure your querySelector is grabbing the right element from the DOM - what is the `id` you're looking for?